lib/checks/landmarks/landmark-is-unique-after.js

```diff
@@ -0,0 +1,30 @@
+const uniqueLandmarks = [];
+return results.filter(currentResult => {
+	const findMatch = someResult => {
+		return (
+			currentResult.data.role === someResult.data.role &&
+			currentResult.data.label === someResult.data.label
+		);
+	};
+
+	const findIndex = (array, matchingFunction) => {
+		for (let index = 0; index < array.length; index++) {
+			if (matchingFunction(array[index])) {
+				return index;
+			}
+		}
+		return -1;
+	};
+
+	const foundResultIndex = findIndex(uniqueLandmarks, findMatch);
+	if (foundResultIndex === -1) {
+		uniqueLandmarks.push(currentResult);
+		currentResult.relatedNodes = [];
```

Comment:
perhaps setting the relatedNodes to an empty array in the check itself, can help negate this step here?

Reply:
I followed the accesskey pattern here; this is because I need info from the node itself (and as such the relatedNodes includes itself when we get to after).

Let me know if this makes sense or I am missing something here. 😄